Week1/github.js

```diff
@@ -0,0 +1,94 @@
+/* global createAndAppend, fetchJSON */
+'use strict';
+{
+    const URL_Rep = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    function renderRepository(container, repository) {
+        const dateOfRepository = new Date(repository.updated_at);
+        container.innerHTML = "";
+        const innerTable = createAndAppend("table", container);
```

Comment:
What is the meaning of this prefix `inner`? I don't see an `outer` table, which would justify the prefix `inner`.